charts/nri-bundle-legacy/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
+description: "DEPRECATED: A Helm chart to deploy New Relic integrations bundled together"
+name: nri-bundle
+version: 3.6.2
+home: https://github.com/newrelic/helm-charts
+icon: https://newrelic.com/assets/newrelic/source/NewRelic-logo-square.svg
+deprecated: true
```

Comment:
should we deprecate this as well? 
Should we deprecate it even if we plan from time to time to provide security updates for a while.